### PR TITLE
Start documentation and cleanup example configuration

### PR DESCRIPTION
**Pull request by gerritholl**
Cleanup the example config file.  Removed variables that did not appear to be used anymore, as confirmed by ``find . -name '*.py' -exec grep 'tle_dirs' .`` along with ``git log -G tle_dirs``.  Add variables that do appear to be used but do not appear in example config, based on reading the source code for ``viirs_dr_runner.py``.  Try to explain the remaining configuration files to the best of my ability, as far as I understand them.

Also starts a first attempt of documentation, so far just in the `README.md` in the package core directory.

### CODE REVIEW

File: examples/viirs_dr_config.cfg_template [hunk 1]
```diff
@@ -1,41 +1,74 @@
...
+log_rotation_backup = 10
+
+# script to update LUTs (must be in PATH)
+mirror_jpss_luts = sdr_luts.sh
+
+# script to update ancillary data (must be in PATH)
+mirror_jpss_ancillary = sdr_ancillary.sh
+
+# ???
+granule_time_tolerance = 10
```
Comment (adybbroe) [inline examples/viirs_dr_config.cfg_template:63]: Maybe add someting along what I just explained on Slack?

**Review by adybbroe — COMMENTED** [review]
LGTM, just one small thing.

File: examples/viirs_dr_config.cfg_template (hunk 1 above)
Comment (gerritholl) [reply examples/viirs_dr_config.cfg_template:65]: Updated with explanation.

**Review by adybbroe — APPROVED** [review]
Thanks a lot @gerritholl for this cleaning up!

**Pull request merged**